scripts/analysis/gcovr_summary.py

```
@@ -99,6 +99,15 @@ def _coverage_emoji(coverage_pct: float) -> str:
         return "🔴"
 
 
+def _short_path(file_path: str) -> str:
+    """冗長な前パス（/home/runner/work/... 等）を削り src/... 形式で表示する。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)